designer/client/src/stylesheets/editor.scss

```
@@ -245,6 +247,20 @@ p code {
       content: "⟩";
     }
   }
+
+  .fileUploadIconContainer {
```

Can we stick with [BEM](https://getbem.com) for naming conventions?

```suggestion
  .file-upload {
```

---

Done